src/EditorFeatures/CSharp/AutomaticCompletion/Sessions/CurlyBraceCompletionSession.cs

```diff
@@ -224,6 +224,19 @@ public override AdjustNewLinesOperation GetAdjustNewLinesOperation(SyntaxToken p
                 return base.GetAdjustNewLinesOperation(previousToken, currentToken, optionSet, nextOperation);
             }
 
+            public override void AddAlignTokensOperations(List<AlignTokensOperation> list, SyntaxNode node, OptionSet optionSet, NextAction<AlignTokensOperation> nextOperation)
```

Comment:
@Pilchie If we dont do it then the closing brace will indent like 'Smart Indenting'(i.e. Normal nested indentation). Block Indentation will expect the 'closing brace' to be at the same indentation as the opening brace. That is why we need to align the braces at the same indentation.